src/utils.js

```diff
@@ -213,6 +216,14 @@ export function isMultiSelect(schema) {
   return Array.isArray(schema.items.enum) && schema.uniqueItems;
 }
 
+export function isFixedItems(schema) {
+  return Array.isArray(schema.items) && schema.items.length > 0 && schema.items.every(item => isObject(item));
+}
+
+export function allowAdditionalItems(schema) {
+  return isObject(schema.additionalItems);
+}
```

Comment:
> Maybe we need a input widget for "any" or "raw"?

Well, that's quite impossible, even if we decide to render a textarea it's unlikely to be able to reflect *any* JSON data structure, so we're stuck here.

Maybe we should simply raise a warning in such a case, and document this limitation in an appropriate section of the documentation? What do you think?